src/stackanalysismodule.ts

```diff
@@ -128,7 +128,7 @@ export module stackanalysismodule {
     uri = null
   ) => {
     let effectiveF8Var: string, argumentList: string;
-    Apiendpoint.API_ECOSYSTEM = ecosystem;
+    Config.API_ECOSYSTEM = ecosystem;
```

Comment:
What is the point of having this global var? This could be removed by passing ecosystem to stackanalysismodule.stackAnalysesLifeCycle

Reply:
I see that ecosystem stored in global variable to pass around. This shall be solved by argumenting it to downstream functions.